Tchap/Modules/PublicRooms/PublicRoomsDataSource.swift

```diff
@@ -116,7 +116,7 @@ final class PublicRoomsDataSource: NSObject {
     // MARK: - Private
     
     private func setupTableViewCells() {
-        self.tableView?.register(PublicRoomTableViewCell.nib(), forCellReuseIdentifier: PublicRoomTableViewCell.defaultReuseIdentifier())
+        self.tableView?.register(PublicRoomsCell.nib(), forCellReuseIdentifier: PublicRoomsCell.defaultReuseIdentifier())
```

Comment:
You could use `self.tableView?.register(PublicRoomTableViewCell.self)` if PublicRoomTableViewCell conform to NibReusable